pygeo/parameterization/DVGeoMulti.py

```diff
@@ -1195,6 +1202,10 @@ def __init__(
         # flag to include feature curves in ID-warping
         self.incCurves = includeCurves
 
+        # list of curves that allow nodes to slide on them. we only use these for the projection step,
+        # but they are not included in the first line based IDWarp
```

Comment:
What does "first" here mean? Also, would it be more accurate to say these curves are not included as seeds in the curve-based deformation?

Reply:
Again, I will defer to @anilyil on this.